pkg/containers/containers.go

```diff
@@ -46,20 +42,23 @@ type CgroupInfo struct {
 
 // New initializes a Containers object and returns a pointer to it.
 // User should further call "Populate" and iterate with Containers data.
-func New(sockets cruntime.Sockets, mapName string, debug bool) (*Containers, error) {
+func New(
+	defCgroup cgroup.Cgroup,
```

Comment:
ditto, the "default" concept should be irrelevant for the reading here (and we can make the "default" logic a bit simpler IMO i'll expand upon it further in the review).